robotcode/src/main/java/org/usfirst/frc/team1885/robot/modules/DriveTrain.java

```diff
@@ -109,7 +109,7 @@ private void setMode(DriveMode mode) {
 			desiredLeftPower = 0;
 			desiredRightPower = 0;
 			setMotorMode(TalonControlMode.PercentVbus);
-			setVoltageRampRate(DEFAULT_RAMP_RATE);
+			//setVoltageRampRate(DEFAULT_RAMP_RATE);
```

Comment:
Voltage Ramp Rate is a safety protection for certain conditions.  It must be present in the drive train code.